### PR TITLE
Fix #6: Added HandMesh

### LINKED ISSUE #6: Setup Player controller
Add ARVRController as child of PlayerOrigin inside Player scene.

The final Player scene will be like this:

- Player (RigidBody)
  - PlayerOrigin (ARVROrigin)
    - PlayerCamera (ARVRCamera)
    - LeftHandController (ARVRController)
       - LeftHandMesh (MeshInstance)
       - LeftHandCollisionShape (CollisionShape) 
    - RightHandController (ARVRController)
       - RightHandMesh (MeshInstance)
       - LeftHandCollisionShape (CollisionShape) 
  - PlayerCollisionShape (CollisionShape)

### PR DESCRIPTION
**Pull request by ilariacuccaro**
Fix #6 
Hand's CollisionShape generates Warning, because CollisionShape should be used as a child of Area, RigidBody, StaicBody... However, the game starts.

### CODE REVIEW

File: Player.tscn [hunk 1]
```diff
@@ -9,5 +41,29 @@
...
+[node name="LeftHandMesh" type="MeshInstance" parent="PlayerOrigin/LeftHandController"]
+transform = Transform( -1, 0, -8.74228e-008, 0, 1, 0, 8.74228e-008, 0, -1, 0, 0, 0 )
+mesh = SubResource( 1 )
+material/0 = null
+
+[node name="CollisionShape" type="CollisionShape" parent="PlayerOrigin/LeftHandController"]
+transform = Transform( 0.2, 0, 0, 0, 0.2, 0, 0, 0, 0.2, 0, 0, -0.0223315 )
+shape = SubResource( 2 )
+
+[node name="RightHandController" type="ARVRController" parent="PlayerOrigin"]
```
Comment (Girbons) [inline Player.tscn:54]: set `controller_id=2` for `RightHandController`

as stated here https://docs.godotengine.org/en/3.1/classes/class_arvrcontroller.html#property-descriptions
```A controller id of 0 is unbound and will always result in an inactive node. Controller id 1 is reserved for the first controller that identifies itself as the left hand controller and id 2 is reserved for the first controller that identifies itself as the right hand controller.```

File: Player.tscn [hunk 2]
```diff
@@ -9,5 +41,29 @@
 [node name="PlayerCamera" type="ARVRCamera" parent="PlayerOrigin"]
 transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1.85, 0 )
```
Comment (Girbons) [inline Player.tscn]: replace `1.85` with `0`

File: Player.tscn [hunk 3]
```diff
@@ -9,5 +41,29 @@
...
+
+[node name="CollisionShape" type="CollisionShape" parent="PlayerOrigin/LeftHandController"]
+transform = Transform( 0.2, 0, 0, 0, 0.2, 0, 0, 0, 0.2, 0, 0, -0.0223315 )
+shape = SubResource( 2 )
+
+[node name="RightHandController" type="ARVRController" parent="PlayerOrigin"]
+transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0.1, 0, 0 )
+
+[node name="RightHandMesh" type="MeshInstance" parent="PlayerOrigin/RightHandController"]
+transform = Transform( -1, 0, -8.74228e-008, 0, 1, 0, 8.74228e-008, 0, -1, 0, 0, 0 )
```
Comment (Girbons) [inline Player.tscn]: set `RightHandMesh` rotation to `-148`

File: Player.tscn [hunk 4]
```diff
@@ -9,5 +41,29 @@
 [node name="PlayerCamera" type="ARVRCamera" parent="PlayerOrigin"]
 transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1.85, 0 )
 
+[node name="LeftHandController" type="ARVRController" parent="PlayerOrigin"]
+transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, -0.1, 0, 0 )
+
+[node name="LeftHandMesh" type="MeshInstance" parent="PlayerOrigin/LeftHandController"]
```
Comment (Girbons) [inline Player.tscn:45]: set `LeftHandMesh` rotation to `148`

**Review by Girbons — CHANGES_REQUESTED** [review]

**Comment by ilariacuccaro** [comment]
Fix #6 
Modify HandMesh rotation and ARVRCamera height

**Review by MaySantucci — APPROVED** [review]

**Pull request merged**